Source/WebCore/accessibility/AccessibilityScrollView.cpp

```diff
@@ -338,15 +338,20 @@ AccessibilityObject* AccessibilityScrollView::parentObject() const
     if (!cache)
         return nullptr;
 
-    WeakPtr owner = m_frameOwnerElement.get();
+    Element* parentElement = m_frameOwnerElement.get();
     if (RefPtr localFrameView = dynamicDowncast<LocalFrameView>(m_scrollView.get()))
-        owner = localFrameView->frame().ownerElement();
+        parentElement = localFrameView->frame().ownerElement();
     else if (RefPtr remoteFrameView = dynamicDowncast<RemoteFrameView>(m_scrollView.get()))
-        owner = remoteFrameView->frame().ownerElement();
-
-    if (owner && owner->renderer())
-        return cache->getOrCreate(*owner);
-    return nullptr;
+        parentElement = remoteFrameView->frame().ownerElement();
+
+    AccessibilityObject* parentAccessibilityObject = nullptr;
```

Comment:
Maybe `ancestorAccessibilityObject`, since we may end up traversing past our direct parent? And `ancestorElement` over `parentElement` too?